Cargo.toml

```diff
@@ -11,10 +11,10 @@ license = "MIT OR Apache-2.0"
 categories = ["data-structures", "encoding"]
 
 [dependencies]
-cid = { version = "0.11.0", features = ["serde-codec"] }
+ipld-core = { version = "0.3.0", features = ["serde"] }
+ipld-extract-links = "0.1.0"
```

Comment:
I'd consider moving this to the `serde` module in `ipld-core`. It doesn't pull in any additional dependencies anyways.

Reply:
That's a a good idea!

Reply:
It does pull in `serde_bytes`. But that's totally fine to have (it's likely somewhere in your dependency tree anyway).

Reply:
https://github.com/ipld/rust-ipld-core/pull/7

Reply:
LGTM!